drivers/storage/portworx/util/util.go

```diff
@@ -326,6 +330,17 @@ func IsHostPidEnabled(cluster *corev1.StorageCluster) bool {
 	return err == nil && enabled
 }
 
+// IsAutoTLSEnabled returns true if the AutoTLS-annotation is true value, and cluster version qualifies
+func IsAutoTLSEnabled(cluster *corev1.StorageCluster) bool {
+	enabled, err := strconv.ParseBool(cluster.Annotations[AnnotationAutoTLS])
```

Comment:
@zoxpx what do we want the default behavior to be if the annotation is not specified?
If this option is not specified, what would the non-auto way for the user to setup TLS?

Reply:
I'll try to re-work the "trigger" -- I was thinking, if someone requests SSL/TLS, but they _do not_ supply their own certificates -- we'd set up with "automatic SSL certs".

I think the current `tls` -block in StorageCluster YAML will allow this.